ci-operator/config/openshift/knative-eventing-contrib/openshift-knative-eventing-contrib-release-v0.17.1.yaml

```diff
@@ -0,0 +1,156 @@
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/camel-source-controller
+  to: knative-eventing-sources-camel-source-controller
+- dockerfile_path: openshift/ci-operator/knative-images/kafka-source-controller/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/kafka-source-controller
+  to: knative-eventing-sources-kafka-source-controller
+- dockerfile_path: openshift/ci-operator/knative-images/event_display/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/event_display
+  to: knative-eventing-sources-event-display
+- dockerfile_path: openshift/ci-operator/knative-images/websocketsource/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/websocketsource
+  to: knative-eventing-sources-websocketsource
+- dockerfile_path: openshift/ci-operator/knative-images/appender/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/appender
+  to: knative-eventing-sources-appender
+- dockerfile_path: openshift/ci-operator/knative-images/kafka-source-adapter/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/kafka-source-adapter
+  to: knative-eventing-sources-kafka-source-adapter
+- dockerfile_path: openshift/ci-operator/knative-images/heartbeats/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/heartbeats
+  to: knative-eventing-sources-heartbeats
+- dockerfile_path: openshift/ci-operator/knative-images/kafka-channel-webhook/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/kafka-channel-webhook
+  to: knative-eventing-sources-kafka-channel-webhook
+- dockerfile_path: openshift/ci-operator/knative-images/github-source-controller/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/github-source-controller
+  to: knative-eventing-sources-github-source-controller
+- dockerfile_path: openshift/ci-operator/knative-test-images/kafka_performance/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/kafka_performance
+  to: knative-eventing-sources-test-kafka-performance
+- dockerfile_path: openshift/ci-operator/knative-test-images/kafka-publisher/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/kafka-publisher
+  to: knative-eventing-sources-test-kafka-publisher
+promotion:
+  additional_images:
+    knative-eventing-contrib-src: src
+  name: knative-0.17.1
```

Comment:
We've been using this format: `knative-v0.17.1` (mind the `v`)

Reply:
Ah! check! 

fixed!